bitnami/minio/2023/debian-11/rootfs/opt/bitnami/scripts/minio/setup.sh

```diff
@@ -33,15 +24,26 @@ minio_regenerate_keys
 if is_boolean_yes "$MINIO_SKIP_CLIENT"; then
     debug "Skipping MinIO client configuration..."
 else
-    if [[ "$MINIO_SERVER_SCHEME" == "https" ]]; then
-        [[ ! -d "${MINIO_CLIENT_CONF_DIR}/certs" ]] && mkdir -p "${MINIO_CLIENT_CONF_DIR}/certs"
-        [[ -d "${MINIO_CERTS_DIR}/CAs" ]] && cp -r "${MINIO_CERTS_DIR}/CAs/" "${MINIO_CLIENT_CONF_DIR}/certs/CAs"
-    fi
     # Start MinIO server in background
     minio_start_bg
     # Ensure MinIO Client is stopped when this script ends.
     trap "minio_stop" EXIT
 
+    # set client env
```

Comment:
Could you explain the reason for moving this logic after minio starts?

Reply:
Because minio will compare all MINIO_* env, it will cause server env not match error when set client env in front.